packages/honeycomb/Button/Button.tsx

```diff
@@ -1,29 +1,25 @@
-import { ReactNode } from "react";
 import clsx from "clsx";
 import styles from "./Button.module.css";
 
-export const Button = ({
-  children,
-  intent,
-  size,
-  mood,
-  onClick,
-}: {
-  children: ReactNode;
+export interface ButtonProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  children: React.ReactNode;
   onClick: () => void;
   intent?: string;
   size?: string;
   mood?: string;
-}) => (
+}
+
+export const Button = (props: ButtonProps) => (
```

Comment:
```suggestion
export const Button = ({className, ...props}: ButtonProps) => (
```
Best to destructure the `className` here and spread that after line 20. This way any `className` provided will be additional to the existing styles and won't override.

Unlikely for our current situation, but good practice.

Reply:
And `children`, generally it's best to destructure any props you specifically need to use in the component, for clarity.